wgpu-core/Cargo.toml

```diff
@@ -88,6 +88,7 @@ replay = ["serde", "naga/deserialize"]
 # --------------------------------------------------------------------
 
 ## Enable creating surfaces using raw-window-handle
+# TODO: This is now already "always-on" via wgpu-types
```

Comment:
Note: this is fine - previously firefox didn't consume RWH, but I just checked and they have audits for the last 3 versions or so, we should probably just make this an unconditional dependency.